create_functions.sql

```diff
@@ -417,56 +510,170 @@ BEGIN
             acc.slot,
             acc.write_version,
             acc.txn_signature
-        FROM account_audit AS acc
+        FROM public.account_audit AS acc
         WHERE
             acc.slot <= in_slot
             AND acc.pubkey = in_pubkey
-        ORDER BY
-            acc.slot DESC, acc.write_version DESC
+        ORDER BY 
```

Comment:
ORDER BY acc.pubkey DESC is unnecesary

Reply:
I investigated query with EXPLAIN command and found out that most performant sorting is possible only when every column from compound key is present in sort condition - then no SeqScans appears in execution plan.